src/relay/op/tensor/transform.cc

```diff
@@ -2668,8 +2668,9 @@ Array<Array<Layout>> StridedSliceInferCorrectLayout(const Attrs& attrs,
       params->begin = new_begin;
       params->end = new_end;
     }
+    return InferCorrectLayoutOutput({{layout}, {layout}}, Attrs(params));
   }
-  return {{layout}, {layout}};
+  return InferCorrectLayoutOutput({{layout}, {layout}}, attrs);
```

Comment:
`return out_default;`?

Reply:
`out_default` is `{{Layout::Undef()}, {Layout::Undef()}}`, here we do want to return non-null layout. I don't know if this distinction is intentional, but I think `Layout::Undef()` implies failure while `{{layout}, {layout}}` implies success.

Reply:
nvm, I probably just messed up myself when reading this part.